entrypoint.sh

```diff
@@ -22,4 +20,4 @@ cd "${WORKING_DIRECTORY:-.}"
 echo "Building the gem"
 find . -name '*.gemspec' -maxdepth 1 -exec gem build {} \;
```

Comment:
Isn't the idea to remove the maxdepth here for monorepos?

Reply:
I had to update the exec  with `dirname {}` and `basename {}`